test/src/test/java/hudson/model/JobPropertyTest.java

```diff
@@ -115,7 +133,13 @@ public JobPropertyWithConfigImpl(String name) {
         }
 
         @TestExtension("configRoundtrip")
-        public static class DescriptorImpl extends JobPropertyDescriptor {}
+        public static class DescriptorImpl extends JobPropertyDescriptor {
+            @Override
+            public JobProperty<?> newInstance(StaplerRequest req, JSONObject formData) throws FormException {
```

Comment:
I do not think we lose any coverage. In the retained original portion of the test case, the `super` call results in a nonempty `name`, so the override is a no-op. We only change the behavior in the override for the newly introduced latter half of the test case.

I mean, we lose coverage for any code calling `Util.isOverridden` on this `newInstance` overload, but I do not think there is any.